drive_cli/dcli.py

```diff
@@ -27,23 +27,24 @@
 def login(remote):
     try:
         token = os.path.join(dirpath, 'token.json')
```

Comment:
This method would never lead to an exception. I'm not sure why are you trying to handle this instead of [this](https://github.com/nurdtechie98/drive-cli/blob/d2aca6bd12d8dc7634a0194b7825ea014bd4bab0/drive_cli/dcli.py#L29).